Makefile

```diff
@@ -395,6 +374,18 @@ define buildpage
 		--var "staging"="$(DEPLOY_TARGET)" $< > $@
 endef
 
+define applypatches
+	git apply --directory=src scripts/fastclick.patch;
+	git apply --directory=src/lib scripts/slipjs.patch;
```

Comment:
If the above comment is done, this could be simple function where we pass a library name (applypatch(slipjs))